### ISSUE TITLE
0 converted to empty string during URL construction

### ISSUE THREAD

**Issue by barun-saha** [ierror/django-js-reverse#41]
First of all, django-js-reverse is amazing! The JS files now look more readable because of it.

However, recently, a strange issue came up while passing 0 (zero) as an argument to the URL -- it got translated to an empty string. In particular, I have this URL pattern:
```python
url(r'^plot/hop_cnt_seq_num/(?P<src_node>\d+)/(?P<src_port>\d+)/(?P<dst_node>\d+)/(?P<dst_port>\d+)/$', 'hop_count_pkt_seq_num', name='trace_hop_cnt',)
```

When I invoke 
```js
Urls.trace_hop_cnt(src_node, src_port, dst_node, dst_port)
```
from within JS with the tuple (66, 0, 67, 0), the resulting URL becomes `/plot/hop_cnt_seq_num/66//67//`, which actually should be `/plot/hop_cnt_seq_num/66/0/67/0/`. However, when invoked with (66, 1, 67, 1), it generates the correct URL `/plot/hop_cnt_seq_num/66/1/67/1/`. So somehow zeroes are being treated otherwise.

Is there any work-around for this? All other URLs are handled fine. I can manually generate the URL (like I was doing previously) as
```js
['/plot/hop_cnt_seq_num', src_node, src_port, dst_node, dst_port].join('/') + '/'
```

that works properly.

**Comment by ierror** [comment]
Thank you for reporting. I found the problem:

https://github.com/ierror/django-js-reverse/blob/develop/django_js_reverse/templates/django_js_reverse/urls_js.tpl#L72

`0 || '' => ''`

There are also cases like 

`0.000 || '' => ''`

i need to think about. **toString** could be a solution.
I'll write some test cases and hope to release a fix soon.

**Comment by ierror** [comment]
As a temporary solution you could use a string instead of number **0**

`Urls.trace_hop_cnt(66, '0', 67, '0')`

**Comment by ierror** [comment]
fixed, i released v0.7.1 to pypi

**Comment by barun-saha** [comment]
@ierror Thanks for your response and the quick fix! I installed the latest version from PyPi and found it working as desired.
